__tests__/fixtures1.blocks/BlockWithDeclaredMix/BlockWithDeclaredMix.js

```diff
@@ -0,0 +1,8 @@
+import { decl } from 'bem-react-core';
+
+export default decl({
+    block : 'BlockWithDeclaredMix',
+    mix() {
```

Comment:
I don't understand difference between `mix` and `addMix`. Which one I need to use in declaration?